### PR TITLE
Fixes #30835 - ovirt Filter SD by data and volume

### PR DESCRIPTION
**Pull request by shiramax**
Depend on: https://github.com/fog/fog-ovirt/pull/96

### CODE REVIEW

**Review by tbrisker — DISMISSED** [review]
This will need a new fog-ovirt release to work, but once that's done and we update to it this pr will be good to go.

**Review by tbrisker — DISMISSED** [review]
Thanks @shiramax !

File: app/models/compute_resources/foreman/model/ovirt.rb [hunk 1]
```diff
@@ -230,7 +230,7 @@ def available_storage_domains(cluster_id = nil)
     end
 
     def storage_domains(opts = {})
-      client.storage_domains({:role => 'data'}.merge(opts))
+      client.storage_domains({:role => %w'data volume'}.merge(opts))
```
Comment (evgeni) [inline app/models/compute_resources/foreman/model/ovirt.rb]: I find `%w'a b'` for `['a', 'b']` highly unreadable. can this be `%w(data volume)` or `['data', 'volume']`?

but maybe that's just me :)

Comment (shiramax) [reply app/models/compute_resources/foreman/model/ovirt.rb]: @evgeni sure, changed it.

**Review by evgeni — APPROVED** [review]

**Pull request merged**